db/seeds.rb

```diff
@@ -5,3 +5,37 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.destroy_all
```

Comment:
@Gataniel why this data in seed? You should put in seed only data which are required for properly application work.